extensions/youtube-downloader/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "youtube-downloader",
-  "title": "YouTube Downloader",
-  "description": "Download YouTube video from Raycast",
+  "title": "Video Downloader",
+  "description": "Download videos from YouTube, X, etc uses yt-dlp",
```

Comment:
Download videos from YouTube, Twitter, Twitch, Instagram and more using yt-dlp CLI